tests/e2e/conftest.py

```diff
@@ -44,3 +44,13 @@ def gen_random_file(location: Union[str, Path], name: Optional[str] = None) -> P
     file = location / name
     file.write_text(str(uuid.uuid4()))
     return file
+
+
+def gen_ing_extr_strategies_grid() -> List[Tuple[str, str, str, bool]]:
+    grid = []
+    for src_type in ("gcp", "aws"):
+        for dst_type in ("local",):  # "storage:", ): # need to release new img of NE
+            for archive_extension in ("tar.gz", "tgz", "zip", "tar"):
+                for extract in (True, False):
+                    grid.append((src_type, dst_type, archive_extension, extract))
+    return grid
```

Comment:
Style comment: usually, tests are written "dirty": using copy-paste and minimum logic in order to ease debugging. [Example](https://github.com/neuromation/platform-api/blob/62ba3301f43eb37e28f3083d7421545c16b4880d/tests/unit/test_job_rest_validator.py#L155). Thus, I'd like us to drop this function and have a large list of hard-coded values instead.

Reply:
but let's do it in a following-up PR